place/supplier_place.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+package protobuf;
+
+option go_package = "/model";
+
+message SupplierPlaceRequest {
+  string id = 1;
+}
+
+message SupplierPlaceResponse {
+  PlaceResponse response = 1;
+}
+
+message PlaceResponse {
```

Comment:
seems like a lot of missing fields from FS response? ex. category, social media, etc

Reply:
will add more as time goes on

Reply:
dont we need all that in the initial version?